README.md

```diff
@@ -26,6 +26,10 @@ Install using pip
 
 Run `curator --help` for usage specifics.
 
+### Common Issues
```

Comment:
Rather than make users feel like there's "common issues," perhaps this would be better titled, **Frequently Asked Questions**